content/literature/computing.Rmd

```diff
@@ -0,0 +1,32 @@
+---
+title: "Scientific computing and data handling workflows"
+date: 2018-03-21
+link-citations: yes
+csl: ../inbo.csl
+bibliography: reproducible_research.bib
+---
+
+This page lists **selected** literature and online resources. Some are related to existing tutorial pages, while others are not. They are supposed to be of high interest to this site's users.
+
+Several of the resources were added based on an inspiring
+[talk](https://docs.google.com/presentation/d/10KkXEv4r3wWtdKvB6RFOPe809eMNldODaRqQxn-jeME/edit?usp=sharing) 
+by Julia Lowndes at the SAFRED conference, Brussels, 27 Feb 2018.
+
+## Overviews {#overviews}
+
+- @wilson_good_2017: set of **good computing practices** that every researcher can adopt
+- @british_ecological_society_guide_2014: planning the **data life cycle**; creating, processing, documenting, preserving, sharing & reusing data
+- @cooper_guide_2017: file organisation, workflow documentation, **code** reproducibility and readability, writing reproducible reports, version control and code archiving
+- @ibanez_practicing_2014: vision on reproducible science, routine **practices**, collaboration, literate computing
+
+See also some resources related to [learning and education](#educate) and the [discipline of open and reproducible science](#discipline).
```

Comment:
The `#educate` and `#discipline` anchors should be reinserted in `open_science.Rmd` and `skills.Rmd`.

Reply:
solved in a4dace0e